units/src/amount.rs

```diff
@@ -1046,12 +1046,20 @@ impl Amount {
 
     /// Unchecked addition.
     ///
-    /// Computes `self + rhs`.  Panics in debug mode, wraps in release mode.
+    /// Computes `self + rhs`.
+    ///
+    /// # Panics
+    ///
+    /// Panics in debug mode, wraps in release mode.
```

Comment:
In 3f76015ee911a78c9e7cb65953166dad2614cda6:

Can you clarify this to say "**On overflow,** panics in debug mode and wraps in release mode.".

Reply:
Clarified in both `unchecked_add()` and `unchecked_sub()`